### ISSUE TITLE
Documentation error

### ISSUE THREAD

**Issue by tirrorex** [ashleymills/Reachability.swift#339]
"note that the framework name has changed from ReachabilitySwift to Reachability"
It's the other way around, it changed from Reachability to ReachabilitySwift.

**Comment by EdemAttiogbe** [comment]
@tirrorex Actually, I think the documentation is correct.  'Reachability' is the name of the current framework, but 'ReachabilitySwift' is still the name of the Cocoapod, as defined by the podspec file (if you're using Cocoapods)

**Comment by tirrorex** [comment]
@EdemAttiogbe that's weird then, because it specifically said "If you're adding Reachability.swift using CocoaPods, note that the framework name has changed from ReachabilitySwift to Reachability"
I assume it is not deployed yet then ?
Considering adding Reachability instead of ReachabilitySwift  links to the old framework.

Maybe i missed something

**Comment by EdemAttiogbe** [comment]
@tirrorex Yeah, it's little confusing, but when I looked at it further, the reference to 'framework' basically means swift 'module'.  My understanding is that prior to version 4.0, you would have to import 'ReachabilitySwift' instead of 'Reachability' into your source code.  

Version 4.0 and later change the original import of 'ReachabilitySwift' to 'Reachability'.  For example, due my need of using Swift 4.2, I currently use version 4.3.0 of Reachability, and my import statement is `import Reachability`.  

What I noticed (that I originally made of mistake of missing) is that the name of the Cocoapods podspec is still 'ReachabilitySwift' (https://github.com/CocoaPods/Specs/tree/master/Specs/d/d/6/ReachabilitySwift), which means that you have to use pod 'ReachabilitySwift' in your podfile.  Note that this is **_not_** what I believe the author was referring to when stating the framework name changed.  

Does this clear things up?  It could be that I'm missing something to......

**Comment by tirrorex** [comment]
@EdemAttiogbe yes it makes more sense now.
Though i believe the documentation could use some clarification on this ;)
Thanks
